src/app-bar-new.jsx

```diff
@@ -0,0 +1,307 @@
+const React = require('react');
+const Paper = require('./paper');
+const ThemeManager = require('./styles/theme-manager');
+const DefaultRawTheme = require('./styles/raw-themes/light-raw-theme');
+const StylePropable = require('./mixins/style-propable');
+
+const AppBarNew = React.createClass({
+
+	mixins: [StylePropable],
+
+	getInitialState () {
```

Comment:
Honestly, I don't think any of the props from http://material-ui.com/#/components/appbar are used anymore.